test/object-store/util/test_file.cpp

```diff
@@ -235,7 +235,9 @@ static std::error_code wait_for_session(Realm& realm,
     bool completed = cv.wait_for(lock, timeout, [&]() {
         return wait_flag == true;
     });
-    REALM_ASSERT_RELEASE(completed);
```

Comment:
At least one test times out basically 100% of the time so aborting when that happens made the test suite unrunnable.